core-plugins/src/test/java/io/cdap/plugin/batch/aggregator/GroupByRelationalTest.java

```diff
@@ -54,6 +54,7 @@ public class GroupByRelationalTest {
   @Before
   public void setUp() {
     Mockito.doReturn(engine).when(relationalTranformContext).getEngine();
+    Mockito.doReturn(true).when(relationalTranformContext).isFeatureEnabled(Mockito.anyString());
     Mockito.doReturn(Optional.of(expressionFactory)).when(engine).getExpressionFactory(Mockito.any(),
```

Comment:
we don't need this for `dedup aggregator test`?

Reply:
We are not actually testing the `transform` method for the Dedup aggregator. We only test the logic to generate the definition.